tests/units/Extension/Constants.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Wasm\Tests\Units\Extension;
+
+use ReflectionExtension;
+use Wasm\Tests\Suite;
+
+class Constants extends Suite
```

Comment:
Generally `final` unless you really need something to be extensible (applies also to tests)

Reply:
I'm not that strict, while I respect your opinion :-).